src/views/TargetsDetailsView.vue

```diff
@@ -108,19 +108,6 @@
                 <apexchart width="100%" height="270px"  type="bar" :options="optionsBar" :series="seriesBar"></apexchart>

```

Comment:
We have hard conding stuff like this
```
 <div v-bind:style ="{background:LinearGradient}">43</div>
```
check the 43 in Number of subdomains by each open ports